worker/src/main/scala/org/apache/celeborn/service/deploy/worker/profiler/JVMProfiler.scala

```diff
@@ -77,4 +82,24 @@ class JVMProfiler(conf: CelebornConf) extends Logging {
       })
     }
   }
+
+  private def extractionDir: Path = {
+    val extractionDirectory = applicationsDir
+    if (extractionDirectory.toFile.exists()) {
+      extractionDirectory
+    } else {
+      Utils.createTempDir(conf.workerWorkingDir, "profiler").toPath
```

Comment:
can it just be `conf.workerWorkingDir/profiler` instead of temp dir?

Reply:
Creating temp directory could avoid conflict of extracted libraries.

Reply:
maybe we shall use `workerBaseDirs`?